UnityUtils/Scripts/Extensions/ResourcesUtils.cs

```diff
@@ -1,8 +1,11 @@
 ﻿using UnityEngine;
+#if ENABLED_UNITY_URP
 using UnityEngine.Rendering;
+#endif
 
 namespace UnityUtils {
     public static class ResourcesUtils {
+        #if ENABLED_UNITY_URP
```

Comment:
nit. convention: left-align #if, #else, #elif, #endif, and other preprocessor directives (like #region, #pragma) at the start of a line. This ensures that the preprocessor directives are clearly visible and easy to differentiate from regular code.

Reply:
yeah my bad, this is normally my style of convention. i will convert and re-push.